common/economic_categories/hfe_categories_population.txt

```diff
@@ -1,9 +1,116 @@
 #
 
-hfe_job_population_navigator.0 = {
-	parent = planet_jobs
+hfe_job_population_navigators = {
+	parent = ship_upgrades
```

Comment:
That is intentional. Navigators are tied to Star Bases. As there is no Star Base Economic Category, I used the next best thing.

Reply:
So why it should be ship upgrade?

Reply:
As I said, there are no Star Base Economic Categories that would work with this, if any. Since the Navigators' Guild provides improvements to ships, I used that.

Reply:
Do you understand it not just "oh, it's about ship, lets set category about ships", this is about ai budgeting and ai have no consumer goods, etc budget for pop jobs.

Reply:
Okay, done